apa-annotated-bibliography.csl

```diff
@@ -413,7 +496,15 @@
       <else-if type="legal_case">
         <group prefix=" (" suffix=")" delimiter=" ">
           <text variable="authority"/>
-          <date variable="issued" form="text"/>
+	  <choose>
+	    <if variable="container-title" match="any">
+	      <!--Only print year for cases published in reporters-->
+	      <date variable="issued" form="numeric" date-parts="year"/>
+	    </if>
+	    <else>
+              <date variable="issued" form="text"/>
+	    </else>
+	  </choose>
```

Comment:
@rmzelle though looking at this (partly because there are a ton of these, most of them not new) -- I'm wondering if it makes sense to discourage explicitly setting match attributes. I don't think they make the style harder to read (the way single-member groups do, say) and they prevent people from forgetting to set the attribute when adding additional variables.